### ISSUE TITLE
BigQuery: omitting "inner" in a join throws an error

### ISSUE THREAD

**Issue by joffreyvillard** [taozhi8833998/node-sql-parser#739]
**Describe the bug**

In BigQuery (as in many other SQL flavors), `inner` is optional for inner joins (https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#inner_join).

The query below is hence a perfectly valid BigQuery SQL query (executes well in BigQuery SQL Workspace), but  `node-sql-parser` raises the error below (works well if we specify `inner join` instead of `join`):

```
select 
    organization.name,
    count(*) as nb_payments

from product.organization
join product.payment on organization_id = organization.id

group by 1
order by 2 desc
```

Error:

```
[SyntaxError]: Expected [A-Za-z0-9_] but " " found
```

**Database Engine**
BigQuery

**To Reproduce**
-the SQL that be parsed: see above
-the node-sql-parser version: see above: 3.6.10
-the node version: 14.15.5

**Expected behavior**
No error, and the same AST as if `inner` had been written down.

**Screenshots**
-

**Additional context**
-

**Comment by taozhi8833998** [comment]
@joffreyvillard fixed in the `3.7.1` version.

**Comment by joffreyvillard** [comment]
Thanks!
